settings.gradle

```diff
@@ -8,5 +8,6 @@ gradleEnterprise {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
+        publishAlways()
```

Comment:
Can you remove this? We don't publish always because we cannot accept the terms of service on behalf of a contributor

Reply:
How about `publishAlwaysIf` on CI?

Reply:
Yes, that we can do

Reply:
e34b9be69c2f586ed038193558c9740e3c71f6ca